.travis.yml

```diff
@@ -11,6 +11,8 @@ matrix:
           env: KERAS_BACKEND=theano
         - python: 2.7
           env: KERAS_BACKEND=tensorflow
+        - python: 2.7
+          env: KERAS_BACKEND=theano INTEGRATION_TESTS=true
```

Comment:
I'm just afraid that tensorflow is not mature enough. Also there are unittests that are not passing on tensorflow so failure of IT test might be difficult to parse at this stage. Also adoption of tensorflow is probably considerably less than of theano so its better to use theano to be on a safe side and cover the majority of users. 

Also, since its a separate job, it does not create a bottleneck yet (5 min vs 9 min max job) so compilation time is not an issue (yet).

But if you feel strongly for tensorflow I can switch it.